streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java

```diff
@@ -1396,6 +1399,11 @@ private static HostInfo parseHostInfo(final String endPoint) {
      */
     public synchronized void start() throws IllegalStateException, StreamsException {
         if (setState(State.REBALANCING)) {
+            log.debug("Initializing STANDBY tasks for existing local state");
+
+            // initialize a dummy Task for any state we have locally
```

Comment:
@cadonna @mjsax and I had a conversation about naming these in an earlier comment thread. "Pending tasks" was the best name we could come up with so far. I'm open to better names though!

I'm not convinced that "offset reporting tasks" etc. is a good solution, because we actually use these tasks for more than just reporting offsets: we recycle the tasks to become "real" active/standby tasks on-assignment; which is why we called them "pending tasks", as they're tasks we proactively create that are "pending" being used.

Reply:
@cadonna @mjsax 

1. "meta tasks" - since they provide metadata about tasks?
2. "local tasks" - since they reflect the locally available state on-disk?
3. "startup tasks" - since they're used only during startup? <-- this is my favourite so far I think

I'm trying to come up with names that aren't too long, because "changelog offsets reporting tasks" would be a real mouthful, and make variable and method names extraordinarily long!

Reply:
Based on our previous discussion, where @mjsax also suggested "startup tasks" (https://github.com/apache/kafka/pull/16922#discussion_r1733743677) I'm going to go with that and rename everything.

Reply:
Done

Reply:
Ah... The old naming discussion... "startup tasks" work for me... (we just dogged a bullet... 😂)